File: visa-dev-samplecode-master/vdp-c-sharp/vdp-c-sharp/VDPApiTest/visadirect/WatchListScreeningTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Vdp
{
    [TestClass]
    public class WatchListScreeningTest
    {
        private string watchListInquiry;
        private VisaAPIClient visaAPIClient;

        public WatchListScreeningTest()
        {
            visaAPIClient = new VisaAPIClient();
            watchListInquiry =
           "{"
                 + "\"acquirerCountryCode\": \"840\","
                 + "\"acquiringBin\": \"408999\","
                 + "\"address\": {"
                     + "\"city\": \"Bangalore\","
                     + "\"cardIssuerCountryCode\": \"IND\""
                 + "},"
                 + "\"referenceNumber\": \"430000367618\","
                 + "\"name\": \"Mohammed Qasim\""
                 + "}";
        }

        [TestMethod]
        public void TestWatchListInquiry()
        {
            string baseUri = "visadirect/";
            string resourcePath = "watchlistscreening/v1/watchlistinquiry";
            string status = visaAPIClient.DoMutualAuthCall(baseUri + resourcePath, "POST", "Watch List Inquiry Test", watchListInquiry);
            Assert.AreEqual(status, "OK");
        }
    }
}

File: vdp-php/tests/visadirect/WatchListScreeningTest.php
<?php

namespace Vdp;

class WatchListScreeningTest extends \PHPUnit_Framework_TestCase {
	
	public function setUp() {
		$this->visaAPIClient = new VisaAPIClient;
		$this->watchListInquiry = json_encode ([
		  "acquirerCountryCode" => "840",
		  "acquiringBin" => "408999",
		  "address" => [
		    "city" => "Bangalore",
		    "cardIssuerCountryCode" => "IND"
		  ],
		  "referenceNumber" => "430000367618",
		  "name" => "Mohammed Qasim"
		]);
	}
	
	public function testWatchListInquiry() {
		$baseUrl = "visadirect/";
		$resourcePath = "watchlistscreening/v1/watchlistinquiry";
		$statusCode = $this->visaAPIClient->doMutualAuthCall ( 'post', $baseUrl.$resourcePath, 'Watch List Inquiry Test', $this->watchListInquiry );
		$this->assertEquals($statusCode, "200");
	}
}
